Answers.txt

```diff
@@ -2,18 +2,22 @@
 
 Answer - Organizing content to indicate what properties to give to each group of content.
 
-Name two big differences between display: block; and display: inline;.
+
+2.  Name two big differences between display: block; and display: inline;.
```

Comment:
Great answer! Block elements will not always have space above/below, but they will take up the full width of their container. Two block elements next to each other will be on separate lines.